data/schema.rdfa

```diff
@@ -5592,6 +5613,7 @@ Note that Event uses startDate/endDate instead of startTime/endTime, even when d
       <span>Domain: <a property="http://schema.org/domainIncludes" href="http://schema.org/Demand">Demand</a></span>
       <span>Range: <a property="http://schema.org/rangeIncludes" href="http://schema.org/Product">Product</a></span>
       <span>Range: <a property="http://schema.org/rangeIncludes" href="http://schema.org/Service">Service</a></span>
+      <span>Range: <a property="http://schema.org/rangeIncludes" href="http://schema.org/Accommodation">Accommodation</a></span>
```

Comment:
On 19 Jul 2016, at 18:41, vholland <notifications@github.com> wrote:
> 
> In data/schema.rdfa:
> 
> > @@ -5592,6 +5613,7 @@ Note that Event uses startDate/endDate instead of startTime/endTime, even when d
> >        <span>Domain: <a property="http://schema.org/domainIncludes" href="http://schema.org/Demand">Demand</a></span>
> >        <span>Range: <a property="http://schema.org/rangeIncludes" href="http://schema.org/Product">Product</a></span>
> >        <span>Range: <a property="http://schema.org/rangeIncludes" href="http://schema.org/Service">Service</a></span>
> > +      <span>Range: <a property="http://schema.org/rangeIncludes" href="http://schema.org/Accommodation">Accommodation</a></span>
> 
> Why are we adding Accommodation to itemOffered? Shouldn't authors use MTEs (Accommodation and Product/Service as appropriate) for this?
> 

1. Because MTEs do not work (as of now) in Google's toolchain, so anybody adopting this would battle with validation errors.
2. Because we agreed on that design in the conference call a few months ago.
3. Because many of the properties of Product/Service are not really applicable to hotel rooms etc. (e.g. GTIN).

Also, we want to point people to the fact that an offer can include a hotel room etc.

While in GoodRelations, Product was essentially a role notion for Thing, implying that it is the object in an offer, we silently narrowed Product in schema.org to commodities. You can see this also in the fact that Service is not subtype of Product in schema.org, while it is included in the ProductOrService class in GR.